tests/bdd/steps/feature_steps/lms.feature

```diff
@@ -4,7 +4,14 @@ Feature: Steps for our app
     Then the response status code is 200
 
   Scenario: the user has instructor privileges
-    Then the response body does not match 'An instructor needs to launch the assignment to configure it.'
+    Then the response body matches 'role:instructor'
```

Comment:
This feels a little magical without more context/explanation. Would it be possible to create a new step definition that is more clear that we're looking at returned _debug_ information? Otherwise it's a little inscrutable why `role:instructor` would be in the response body.

Reply:
I can't think of a robust way of making the step explain where it's getting this from and have it work. I can sure put in a comment though.